package.json

```diff
@@ -38,11 +38,28 @@
     "auto"
   ],
   "author": "Matteo Collina <hello@matteocollina.com>",
+  "contributors": [
+    {
+      "name": "Frazer Smith",
```

Comment:
TBH I think all or nothing here

Reply:
Is a bit of a pain to go through every repo and get all the contributors so was hoping everyone would add themselves.

Reply:
> Is a bit of a pain to go through every repo and get all the contributors so was hoping everyone would add themselves.

let's keep valid this one, but I would like to check in the npm page what is happning.

Right now, without the `contributors` array, the output shows the npm org team:

<img width="376" alt="image" src="https://github.com/user-attachments/assets/4d7c8d73-9366-4192-a341-6d54ddb12cfc">

Not sure if now it will be overwritten

Reply:
It won't be overwritten, the collaborator list on the npm page is users who have publish privileges. If you look at the [main Fastify package](https://www.npmjs.com/package/fastify) it only shows 4 collaborators even when the collaborators list in the package.json has 20 people.

Reply:
It won't be overwritten, the collaborator list on the npm page is users who have publish privileges. If you look at the [main Fastify package](https://www.npmjs.com/package/fastify) it only shows 4 collaborators even when the collaborators list in the package.json has 20 people.